guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc

```diff
@@ -1,150 +1,79 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="Updating-Disconnected-{project-context}-on-EL9_{context}"]
-= Updating a disconnected {ProjectServer} on {RHEL} 9
+= Updating a disconnected {ProjectServer} on {EL} 9
 
-You can update your disconnected {Project} on {RHEL} 9 by synchronizing the required repositories on the connected {Project} and syncing the content to the disconnected {Project} using `reposync`.
+You can update your disconnected {Project} on {EL} 9 by synchronizing the required repositories on the connected {Project} and syncing the content to the disconnected {Project} by using `syncable` exports.
```

Comment:
```suggestion
You can update your disconnected {Project} on {EL} 9 by synchronizing the required repositories on the connected {Project} and syncing the content to the disconnected {Project} by using syncable exports.
```

suggestion: drop emphasis

it's not a command, so I think no emphasis fits the docs better.